### PR TITLE
Feature/display satisfaction overview

### PR DESCRIPTION
**Pull request by Samox**

### CODE REVIEW

**Comment by Samox** [comment]
screen shot 2017-07-25 at 11 32 22

File: src/components/ui/ListForms.jsx [hunk 1]
```diff
@@ -24,8 +24,9 @@ class ListForms extends PureComponent {
         label = <Label color="red" className="pinned">Red bucket</Label>;
       }
 
+      var formTitle = form.sprint ? `${form.project} - Sprint ${form.sprint}` : `${form.project}`
```
Comment (BrunoGodefroy) [inline src/components/ui/ListForms.jsx]: const

File: src/redux/reducer.js [hunk 2]
```diff
@@ -59,12 +62,31 @@ export default function reducer(state = initialState, action = {}) {
         loading: true,
       };
     case types.FETCH_FORMS.SUCCESS:
+      const forms = action.payload.responses
+      var numberOfWahou = 0
+      var numberOfOK = 0
+      var numberOfKO = 0
```
Comment (BrunoGodefroy) [inline src/redux/reducer.js]: let

**Review by BrunoGodefroy — CHANGES_REQUESTED** [review]
You should use the new `satisfaction` attribute of the form to display the green or red label next to each form

**Review by BrunoGodefroy — APPROVED** [review]

**Pull request merged**